### DISCUSSION TITLE
Request fulfilled notification

### DISCUSSION THREAD

**Discussion by pkmnjourney** [activexray/Doplarr#93] [Ideas]
Are there any plans to notify the user who placed the request when the request is fulfilled? It could privately message the user once the movie, for example, is available for viewing or if the request failed because the movie did not meet the minimum criteria.

**Comment by ddcbeatty** [comment]
I'd love to see this implemented also!

**Comment by thedinz** [comment]
I just came here to say this wasn't working lol, so its not that its not working it doesn't exist? I switched from requestrr to doplarr and requestrr had this feature which is why i assumed it just wasn't working, but now i know, the feature is non existent as of now.
